x11-drivers/nvidia-drivers/nvidia-drivers-525.85.12.ebuild

```diff
@@ -16,13 +16,12 @@ IUSE=""
 S="${WORKDIR}"
 
 RDEPEND="
-	=x11-drivers/nvidia-metadata-${PV}
+  =x11-drivers/nvidia-metadata-${PV}
 "
 
 src_install() {
-  insinto "/usr/share/oem"
-  doins -r "${FILESDIR}/units"
-  exeinto "/usr/share/oem/bin"
+  systemd_dounit "${FILESDIR}/units/nvidia.service"
+  exeinto "/usr/lib/nvidia/bin"
   doexe "${FILESDIR}/bin/install-nvidia"
   doexe "${FILESDIR}/bin/setup-nvidia"
```

Comment:
These files should be added as new files in `/usr`, or? I don't see them in the output of the image changes job.

Reply:
Ah, you need to add `x11-drivers/nvidia-drivers` as dependency of the image ebuild (`coreos-base/coreos/coreos-0.0.1.ebuild` but only for amd64, see end of file)

Reply:
Discovered the same thing again^^ https://github.com/flatcar/coreos-overlay/pull/2160#issuecomment-1335098398

Reply:
Ah, I did add to the ebuild but it was nvidia-metadata, instead of nvidia-drivers.